extension/chrome/elements/compose-modules/compose-footer-module.ts

```diff
@@ -34,7 +34,7 @@ export class ComposeFooterModule extends ViewModule<ComposeView> {
   public createFooterHtml = (footer: string) => {
     const sanitizedPlainFooter = Xss.htmlSanitizeAndStripAllTags(footer, '\n');
     const sanitizedHtmlFooter = sanitizedPlainFooter.replace(/\n/g, '<br>');
-    const footerFirstLine = sanitizedPlainFooter.split('\n')[0];
+    const footerFirstLine = sanitizedPlainFooter.split('\n')[0].replace(/&nbsp;/g, '');
```

Comment:
let's replace it with common space, so it'll be ` ` instead of empty string

Reply:
Thank you - will do.

> And let's add a test which will check that there is no double separator in this case. I think it should be a simple one - you can set user's signature by using configProvider.google.aliases config

I agree - I'll be writing the test now. thank you.

Reply:
Roma, for me the `configProvider.google.aliases` doesn't fit very well on the test so I try to re-use the idea of directly putting a signature with `configProvider.google.acctPrimarySignature` which is great for the test. I hope that is also an acceptable approach.